<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CB1PK7K" failures="1" tests="5" name="MvnSelNg" time="23.105" errors="0" timestamp="2022-12-04T09:10:49 CST">
  <testcase classname="mavenAndTestNG.hrm.login.LoginTests" name="test_Login_Functionality_Is_Working" time="1.476">
    <failure type="java.lang.AssertionError" message="expected [Welcome ggg] but found [Welcome Yoll]">
      <![CDATA[java.lang.AssertionError: expected [Welcome ggg] but found [Welcome Yoll]
at mavenAndTestNG.hrm.login.LoginTests.test_Login_Functionality_Is_Working(LoginTests.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test_Login_Functionality_Is_Working -->
  <testcase classname="mavenAndTestNG.hrm.login.LoginTests" name="verify_user_is_not_able_to_login_invalid_credentials" time="0.555"/>
  <testcase classname="mavenAndTestNG.hrm.login.LoginTests" name="verify_user_is_not_able_to_login_invalid_credentials" time="0.188"/>
  <testcase classname="mavenAndTestNG.hrm.login.LoginTests" name="verify_user_is_not_able_to_login_invalid_credentials" time="0.177"/>
  <testcase classname="mavenAndTestNG.hrm.login.LoginTests" name="verify_user_is_not_able_to_login_invalid_credentials" time="0.175"/>
</testsuite> <!-- MvnSelNg -->
